src/runtime/internal.ts

```diff
@@ -49,20 +50,21 @@ export function parseAcceptLanguage(input: string): string[] {
 
 export function getBrowserLocale(): string | undefined {
   let ret: string | undefined
+  const logger = /*#__PURE__*/ createLogger('getBrowserLocale')
```

Comment:
We tag the logs with `createLogger` for each function, do we need to go that far? 😅 

I think it would be better to have a logger tagged per js/ts module.
I believe we can make debugging more efficient from currently by filtering the logs by module.

Reply:
I agree it seems like a bit much, but most of the logs originally already had the function name in the log itself in each log statement (for example https://github.com/nuxt-modules/i18n/pull/3067/files/a44ce5dbcd40696e2cfd3db47b18464476a5cb45#diff-7755b40274cbd4ca5d224c6ed6953280e589110e9289595aeb246ff4a1c07448L57).

The primary reason for tagging them is so the actual code/logic is easier to read and understand (fewer multiline debug log statements, less word repetition). With these refactors I'm hoping to find patterns for simplification like in https://github.com/nuxt-modules/i18n/pull/3063.

Logging per module/file would definitely make more sense, and I think the fact that we had the function name in the logs probably means that `internal.ts` and utils.ts` are not descriptive enough, which we could solve by splitting these up further.

Reply:
> The primary reason for tagging them is so the actual code/logic is easier to read and understand (fewer multiline debug log statements, less word repetition). With these refactors I'm hoping to find patterns for simplification like in #3063.

Okay, I could understand what you are trying to improve for logging. Let's continue to refactor and improve! :)